Doc/library/subprocess.rst

```diff
@@ -843,6 +855,8 @@ The :mod:`subprocess` module exposes the following constants.
    The new process has a new console, instead of inheriting its parent's
```

Comment:
The title of the current section is " Constants". I checked: *all* constants documented here as specific to Windows. I propose to rename the section to "Windows Constants". Can you please do that?